tests/func/test_update.py

```diff
@@ -176,17 +190,19 @@ def test_update_import_url(tmp_dir, dvc, workspace):
     assert dst.read_text() == "updated file content"
 
 
-def test_update_import_url_no_download(tmp_dir, dvc, workspace):
+@pytest.mark.parametrize("outs_exist", (False, True))
+def test_update_import_url_no_download(
```

Comment:
Should there also be a `test_update_import_no_download` (`import` instead of `import-url`)?

Reply:
Indeed. Added a test.